aiohttp/cookiejar.py

```diff
@@ -82,8 +82,16 @@ def __iter__(self) -> 'Iterator[Morsel[str]]':
     def __len__(self) -> int:
         return sum(1 for i in self)
 
+    def _next_whole_second(self) -> datetime.datetime:
+        """Return current time rounded up to the next whole second."""
+        return (
+            datetime.datetime.now(
+                datetime.timezone.utc).replace(microsecond=0) +
+            datetime.timedelta(seconds=0)
+        )
+
     def _do_expiration(self) -> None:
-        now = self._loop.time()
+        now: datetime.datetime = datetime.datetime.now(datetime.timezone.utc)
```

Comment:
No hint needed. Mypy can see what the function returns. 
```suggestion
        now = datetime.datetime.now(datetime.timezone.utc)
```